src/other/acr/acr-fr.hbs

```diff
@@ -0,0 +1,344 @@
+---
+{
+	"title": "Méthodologie du rapport de conformance à l'accessibilité",
+	"language": "fr",
+	"category": "Autre",
+	"description": "Méthode utilisant une approche itérative afin de produire plusieurs évaluations spécialisées et de produire par la suite un rapport de conformité d'accessibilité.",
```

Comment:
```suggestion
	"description": "Méthode utilisant une approche itérative afin de produire plusieurs évaluations spécialisées et de produire par la suite un rapport de conformité à l'accessibilité.",
```

Reply:
fixed and I replicated the untranslated changes you suggested in the English equivalent page.